packages/manager/modules/pci/src/projects/project/kubernetes/details/service/oidc-provider/update-oidc-provider/template.html

```diff
@@ -0,0 +1,47 @@
+<form name="updateOidcProviderForm" id="updateOidcProvider" data-ng-submit="$ctrl.onUpdateOidcProviderConfirm()">
+    <oui-modal
+        data-on-dismiss="$ctrl.goBack()"
+        data-primary-label="{{:: 'pci_projects_project_kubernetes_details_service_update_oidc_provider_action_update' | translate}}"
+        data-primary-disabled="updateOidcProviderForm.$invalid"
+        data-secondary-action="$ctrl.onUpdateOidcProviderCancel()"
+        data-secondary-label="{{:: 'pci_projects_project_kubernetes_details_service_update_oidc_provider_action_cancel' | translate}}"
+        data-loading="$ctrl.isUpdating"
+        data-heading="{{:: 'pci_projects_project_kubernetes_details_service_update_oidc_provider_title' | translate}}"
+    >
+        <p
+            data-translate="pci_projects_project_kubernetes_details_service_update_oidc_provider_description"
+        ></p>
+
+        <!--OIDC provider URL-->
+        <oui-field
+            data-label="{{::'pci_projects_project_kubernetes_details_service_update_oidc_provider_field_url' | translate}}"
+        >
+            <span data-translate="pci_projects_project_kubernetes_details_service_update_oidc_provider_field_url_extra"></span>
+            <input
+                type="text"
+                class="oui-input"
+                data-ng-init="labelKeyId = 'oidcProviderModelUrl'"
+                data-ng-attr-name="{{labelKeyId}}"
+                data-ng-attr-id="{{labelKeyId}}"
```

Comment:
One time binding for `attr` and `id`
```suggestion
                data-ng-attr-id="{{:: labelKeyId}}"
```

Reply:
I let a response here:
https://github.com/ovh/manager/pull/5860#discussion_r744916538